.github/workflows/manifest.yml

```diff
@@ -74,13 +43,8 @@ jobs:
     runs-on: ubuntu-latest
     needs: updateManifest
     steps:
-      - name: Cancel previous incomplete runs
-        uses: styfle/cancel-workflow-action@0.8.0
-        with:
-          access_token: ${{ github.token }}
```

Comment:
Are we not using caching / cancelling previous runs anymore?

Reply:
We are in some actions, using the new `concurrency` configuration. However, the manifest should probably complete even if it has been superseded, otherwise it could corrupt the manifest file.